src/components/button.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function Button(props) {
+  const { onClick, name } = props;
```

Comment:
Please make sure it accepts `color` and `wide` as an incoming props ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png)).

Optional: You can destructure the incoming props on the argument, and since it is a stateless component, you could use arrow functions:

```js
const Button = ({ onClick, name }) => (
  <button
    className={`btn${name === '0' ? ' btn-zero' : ''}`}
    type="button"
    onClick={onClick}
  >
    {name}
  </button>
);
```
But you'll have to change this to take `color` and `wide` props.

Reply:
Ahhhhhh cool! Noted.